.stickler.yml

```diff
@@ -0,0 +1,10 @@
+# add the linters you want stickler to use for this project
+linters:
+  stylelint:
+    # indicate where is the config file for stylelint
+    config: "stylelint.config.js"
+    
+# PLEASE DO NOT enable auto fixing options
+# if you need extra support from you linter - do it in your local env as described in README for this config
+
+# find full documentation here: https://stickler-ci.com/docs
```

Comment:
Kindly remove this linter, you need to use Microverse's linters without any additional ones.

Reply:
@Krishnabot Hi, you need to submit another request through your dashboard, please.